yaramo/signal.py

```diff
@@ -24,7 +25,13 @@ class SignalFunction(Enum):
     Einfahr_Signal = 0
```

Comment:
This is not specific to this MR, but why is snake case used here? It could simply read `Einfahrsignal`, no?

Reply:
It's the same in PlanPro.